src/formBuilder/conditionalInput.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import {
+  Checkbox,
+  Flex,
+  Input,
+  InputLeftAddon,
+  InputGroup,
+  InputRightAddon,
+} from '@chakra-ui/react';
+
+import FieldWrapper from './fieldWrapper';
+import Paragraphs from './Paragraphs';
+
+const ConditionalInput = props => {
+  const {
+    conditionalLabel,
+    conditionalDesc,
+    label,
+    htmlFor,
+    placeholder,
+    name,
+    localForm,
+    append,
+    prepend,
+    onChange = null,
+    disabled,
+    defaultValue,
+  } = props;
+  const { register, setValue } = localForm;
+  const [checked, setChecked] = useState(false);
+
+  return (
+    <Flex flexDir='Column'>
+      <Checkbox
+        name={`${name}Condition`}
+        size='lg'
+        borderColor='white'
+        colorScheme='blackAlpha'
+        iconColor='red.400'
+        iconSize='2rem'
+        spacing='1rem'
+        onChange={ev => {
```

Comment:
Refactor to a function outside the jsx.

and remove nested ternary.

and we are going to change defaultValue so it can't be a function.